snapshots/windows/windows.go

```diff
@@ -349,73 +352,80 @@ func (s *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 			return fmt.Errorf("failed to create snapshot: %w", err)
 		}
 
-		if kind != snapshots.KindActive {
-			return nil
-		}
-
-		log.G(ctx).Debug("createSnapshot active")
+		log.G(ctx).Debug("createSnapshot")
 		// Create the new snapshot dir
 		snDir := s.getSnapshotDir(newSnapshot.ID)
 		if err = os.MkdirAll(snDir, 0700); err != nil {
-			return err
+			return fmt.Errorf("creating snapshot dir: %w", err)
```

Comment:
`failed to create`?

Also if `err` doesn't mention the path, it would be better to include snDir in this error.

Reply:
This was a consequence of bouncing between ```errors.Wrap()``` and ```fmt.Errorf()``` :smile:. Will fix.

Reply:
I'm pretty sure `os.MkdirAll` always returns a `PathError`. Cheating and checking the source says it either returns `PathError` or calls `os.Mkdir`, and _it_ is documented to always return a `PathError`.

Reply:
added the snDir for good measure.